keps/sig-storage/20190530-pv-health-monitor.md

```diff
@@ -537,6 +538,13 @@ This optional service is out of scope for this KEP but may be introduced in a fu
 This option is not in the main proposal because it is a push-based method while CSI uses pull-based method.
 
 ## Graduation Criteria
+### Alpha
+* Initial feature implementation, including:
+  * External controller volume health monitoring.
```

Comment:
Is there a difference between the features implemented in alpha vs beta?

Reply:
Feature should be the same.  Just add more tests (especially E2E) for beta.